firefox_desktop/explores/newtab_interactions.explore.lkml

```diff
@@ -19,4 +19,174 @@ explore: newtab_interactions {
+        newtab_interactions.sum_sponsored_pocket_impressions,
+        newtab_interactions.sum_sponsored_pocket_saves,
+        newtab_interactions.sum_sponsored_pocket_clicks,
+
+        newtab_interactions.clients_with_pocket_impressions,
+        newtab_interactions.clients_with_pocket_saves,
+        newtab_interactions.clients_with_pocket_clicks,
+        newtab_interactions.clients_with_organic_pocket_impressions,
+        newtab_interactions.clients_with_organic_pocket_saves,
+        newtab_interactions.clients_with_organic_pocket_clicks,
+        newtab_interactions.clients_with_sponsored_pocket_impressions,
+        newtab_interactions.clients_with_sponsored_pocket_saves,
+        newtab_interactions.clients_with_sponsored_pocket_clicks,
+      ]
+      filters: [newtab_interactions.submission_date: "after 2022-07-01"]
+    }
+
+    materialization: {
+      datagroup_trigger: newtab_interactions_v1_last_updated
+      increment_key: newtab_interactions.submission_date
+      increment_offset: 1
+    }
+  }
+
+  aggregate_table: rollup__newtab_interactions_search_per_day {
+    query: {
+      dimensions: [
+        newtab_interactions.submission_date,
+        newtab_interactions.channel,
+        newtab_interactions.country_code,
+        newtab_interactions.search_engine,
+        countries.tier,
+        countries.name,
+        countries.pocket_available_on_newtab,
+      ]
+      measures: [
+        newtab_interactions.visits_with_search,
+        newtab_interactions.visits_with_tagged_search_ad_impression,
+        newtab_interactions.visits_with_tagged_search_ad_click,
+        newtab_interactions.visits_with_tagged_follow_on_search_ad_impression,
+        newtab_interactions.visits_with_tagged_follow_on_search_ad_click,
+        newtab_interactions.visits_with_any_ad_click,
+        newtab_interactions.visits_with_any_ad_impression,
+
+        newtab_interactions.sum_searches,
+        newtab_interactions.sum_tagged_search_ad_impressions,
+        newtab_interactions.sum_tagged_search_ad_clicks,
+        newtab_interactions.sum_tagged_follow_on_search_ad_impressions,
+        newtab_interactions.sum_tagged_follow_on_search_ad_clicks,
+        newtab_interactions.sum_all_search_ad_clicks,
+        newtab_interactions.sum_all_search_ad_impressions,
+
+        newtab_interactions.clients_with_search,
+        newtab_interactions.clients_with_tagged_search_ad_impression,
+        newtab_interactions.clients_with_tagged_search_ad_click,
+        newtab_interactions.clients_with_tagged_follow_on_search_ad_impression,
+        newtab_interactions.clients_with_tagged_follow_on_search_ad_click,
+        newtab_interactions.clients_with_any_ad_click,
+        newtab_interactions.clients_with_any_ad_impression,
+      ]
+      filters: [newtab_interactions.submission_date: "after 2022-07-01"]
+    }
+
+    materialization: {
+      datagroup_trigger: newtab_interactions_v1_last_updated
```

Comment:
just for the benefit of my understanding, where does this live, or how is it generated etc?

Reply:
These are generated in looker-hub by looking at a table's last updated timestamp (usually by ETL). Here's the most relevant doc: https://docs.telemetry.mozilla.org/cookbooks/looker/performance_caching.html, though in the caching context, and the [proposal](https://docs.google.com/document/d/1dPwa5KZnKHzYj3hULHy9unczCXugwr6rZHW1gveQ2fc/edit#heading=h.5x0d5h95i329).